cmd/down.go

```diff
@@ -15,7 +15,19 @@ var downCmd = &cobra.Command{
 }
 
 func downCommand(cmd *cobra.Command, _ []string) {
-	cfg, err := newConfigFromEnv()
+	var fileName *string
+	if cmd.Flags().Changed("file") {
+		fileFlag := &struct {
```

Comment:
Should move this to a function, duplicated between down and up now.

Reply:
Moved to new func checkFileFlag